### PR TITLE
Add overrides for `insights-core` and `pynetbox`

### PR DESCRIPTION
**Pull request by KiruyaMomochi**
- overrides: add insights-core
- overrides: add pynetbox

### CODE REVIEW

File: overrides/default.nix [hunk 1]
```diff
@@ -1946,6 +1946,10 @@ lib.composeManyExtensions [
         }
       );
 
+      pynetbox = super.pynetbox.overridePythonAttrs (old: {
```
Comment (adisbladis) [inline overrides/default.nix:1949]: Why isn't this added in the build systems json file?

Comment (cpcloud) [reply overrides/default.nix:1949]: @KiruyaMomochi Any thoughts here?

Comment (KiruyaMomochi) [reply overrides/default.nix:1949]: Sorry I think I have sent the comment but it have stayed in pending state for one week...

Comment (cpcloud) [reply overrides/default.nix:1949]: Ah, I guess I missed that!

**Pull request merged**